assets/js/payment-method-extensions/payment-methods/stripe/stripe-utils/type-defs.js

```diff
@@ -267,24 +267,28 @@
  * @typedef {Object} Stripe Stripe api object.
  */
 
+/**
+ * @typedef {Object} CCIcon
```

Comment:
I'd prefer a full name `CreditCardIcon` here instead of `CCIcon`. `CCI` can be ambiguous as to it's meaning.